presto-spi/src/main/java/com/facebook/presto/spi/function/OperatorType.java

```diff
@@ -32,7 +32,8 @@
     SUBSCRIPT("[]"),
     HASH_CODE("HASH CODE"),
     SATURATED_FLOOR_CAST("SATURATED FLOOR CAST"),
-    IS_DISTINCT_FROM("IS DISTINCT FROM");
+    IS_DISTINCT_FROM("IS DISTINCT FROM"),
+    IS_NOT_DISTINCT_FROM("IS NOT DISTINCT FROM");
```

Comment:
if we have IS_DISTINCT_FROM, do we need a IS_NOT? can't we just do NOT of the operator?